server/blob/blob-cassandra/pom.xml

```diff
@@ -65,6 +65,10 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>metrics-logger</artifactId>
```

Comment:
Why? MetricFactory API is not enough?

Reply:
https://github.com/apache/james-project/pull/436#discussion_r634214463